src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck is working");
```

Comment:
The message "The truck is working" should be more informative and indicate that a Truck is working. Consider specifying the type of machine in the message, such as "Truck is working".